app/assets/scripts/components/instruction-modal.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+import React from 'react';
+import OpenAQ from 'openaq-design-system';
+var { Modal, ModalHeader, ModalBody, ModalFooter } = OpenAQ.Modal;
+
+var InstructionModal = React.createClass({
+  displayName: 'InstructionModal',
+  propTypes: {
+    open: React.PropTypes.bool
+  },
+
+  getInitialState: function () {
+    return {
+      open: false
+    };
+  },
+
+  closeModal: function () {
+    this.setState({open: false});
+  },
+
+  componentWillReceiveProps: function (nextProps) {
+    this.setState({open: nextProps.open});
```

Comment:
Instead of having the state on the modal, you should pass a callback function that is called on when the modal closes. The parent then takes care of storing the state. This was you avoid duplication.